fitbenchmarking/fitbenchmarking
# Add Gradient Free Optimizers

#### Description of Work

Fixes #723 


#### Testing Instructions

1. Test gradient free optimizers with problems that have finite bounds for all parameters
2. Check docs and tests (stochastic so not included in regression tests)
3.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
